yarn.lock

```diff
@@ -3236,6 +3236,37 @@
     "@types/react" "^16.13.1 || ^17.0.0 || ^18.0.0"
     react-use "^17.2.4"
 
+"@backstage-community/plugin-redhat-argocd-common@1.0.3":
+  version "1.0.3"
+  resolved "https://registry.yarnpkg.com/@backstage-community/plugin-redhat-argocd-common/-/plugin-redhat-argocd-common-1.0.3.tgz#755ab8962c8728225a60fe814eef5bb68d1721c8"
+  integrity sha512-cu/u2kk01ueQX5Jy4J88rjDg+8UyXtl7CsaqNX+XbWNaoCLkuHT0O5oC8V75XXQ+v9K+1hFG5HdVXYX7evN/+g==
+  dependencies:
+    "@backstage/plugin-permission-common" "^0.8.1"
+
+"@backstage-community/plugin-redhat-argocd@1.6.1":
+  version "1.6.1"
```

Comment:
please rebase. latest is 1.6.6

https://github.com/backstage/community-plugins/blob/main/workspaces/redhat-argocd/plugins/argocd/package.json